### ISSUE TITLE
Adam's learning rate

### ISSUE THREAD

**Issue by cahya-wirawan** [rajveermalviya/language-modeling#3]
Hi,

I remember, when I tried your code 2 weeks ago,  that the loss during training didn't really went down even after many epochs. I think this is because you use learning rate of 1.0 in the config file. After I use Adam optimizer with it's default learning rate (which is 0.001), the training get much better. 

Cahya

**Comment by rajveermalviya** [comment]
Thanks for addressing this issue, @cahya-wirawan , I have now updated the config file.
